### PR TITLE
Don't trim predefined slugs

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: resolwe/flow/models/fields.py [hunk 1]
```diff
@@ -110,8 +110,11 @@ def pre_save(self, instance, add):
                 slug = ''
 
         if slug:
-            # Make sure that potentially added sequence won't excede maximal length.
-            slug = slug[:(self.max_length - MAX_SLUG_SEQUENCE_DIGITS - 1)]
+            # Make sure that auto generated slug with added sequence
+            # won't excede maximal length.
+            # Validation of predefined slugs is handled by Django.
+            if not predefined_slug:
```
Comment (kostko) [inline resolwe/flow/models/fields.py]: Can you combine with the if above?

Comment (dblenkus) [reply resolwe/flow/models/fields.py]: No, just trimming has to be skipped for predefined slugs. The rest of the above if is still needed to determine availability of the slug

Comment (kostko) [reply resolwe/flow/models/fields.py]: Ah I didn't see there is code below in the same if, nevermind.

**Comment by tjanez** [comment]
I've run all Tox environments locally and all of them succeeded so I think we can merge this since we don't know how long [Travis CI will need to restore their service](https://www.traviscistatus.com/incidents/c1s4dlyxd4lf).

**Pull request merged**